src/components/Typography/HX/HX.tsx

```diff
@@ -49,9 +49,8 @@ export type HXProps = {
   copyable?: boolean | CopyConfig,
 
   /**
-   * Displays ellipsis when text overflows. By default, the text will be truncated if it
-   * exceeds the length of 2 rows, showing all informational content in a tooltip.
-   * The attribute is further customizable according to the following data model:
+   * Displays ellipsis when text overflows.
```

Comment:
As we discussed, I would add specific roles for heading components (h1, h2, ...)